README.md

```diff
@@ -355,7 +355,7 @@ Tell your users that you adhere to the Conventional Commits specification:
 
 `standard-version` is different because it handles the versioning, changelog generation, and git tagging for you **without** automatic pushing (to GitHub) or publishing (to an npm registry). Use of `standard-version` only affects your local git repo - it doesn't affect remote resources at all. After you run `standard-version`, you still have the ability to review things and correct mistakes if you want to.
 
-They are both based on the same foundation of structured commit messages (using [Angular format](https://github.com/bcoe/conventional-changelog-standard/blob/master/convention.md)), but `standard-version` is a good choice for folks who are not yet comfortable letting publishes go out automatically. In this way, you can view `standard-version` as an incremental step to adopting `semantic-release`.
+They are both based on the same foundation of structured commit messages, but `standard-version` is a good choice for folks who are not yet comfortable letting publishes go out automatically. In this way, you can view `standard-version` as an incremental step to adopting `semantic-release`.
```

Comment:
I don't know that someone necessarily wants to move towards fully automated releases, as an example here's what we're doing currently at Google:

https://github.com/googleapis/release-please/pull/211

We create candidate PRs which an engineer can eventually land, perhaps we could tweak this wording a bit talking about this as a spectrum of automation that someone might have:

* manual work.
* release PRs like release please.
* continuous releases.

Reply:
I've reworded this a bit, let me know what you think!

Taking a step back, it looks like the comparison to `semantic-release` was added as part of the FAQ ~3 years ago... thinking about the expanded support for `bumpFiles` and movement toward the config-spec, I wonder if the README as a whole needs some finessing and if we could work toward dropping this section altogether. I _do_ see value in having a section devoted to common implementations/patterns to help folks better understand how "we" fit into the ecosystem, but maybe we can do it without specific comparisons.